wolfcrypt/src/ecc.c

```diff
@@ -4198,6 +4198,9 @@ int wc_ecc_shared_secret(ecc_key* private_key, ecc_key* public_key, byte* out,
    !defined(WOLFSSL_ATECC608A)
    CRYS_ECDH_TempData_t tempBuff;
 #endif
+
```

Comment:
This function looks like it can hit the end without a return.

Reply:
It does but fixed to be clearer.

Reply:
It does but fixed to be clearer.